templates/repo/view_file.tmpl

```diff
@@ -1,12 +1,8 @@
 <div class="{{TabSizeClass .Editorconfig .FileName}} non-diff-file-content">
 	<h4 class="file-header ui top attached header">
-		<div class="file-header-left">
+		<div class="file-header-left df aic">
 			{{if .ReadmeInList}}
-				{{if .FileIsSymlink}}
-					<i class="icons"><i class="book icon"></i><i class="bottom left corner tiny inverted share icon"></i></i>
-				{{else}}
-					<i class="book icon"></i>
-				{{end}}
+				{{svg "octicon-book" 16 "mr-2"}}
```

Comment:
This was rendering a book with a overlayed link in case a readme was a symlink. Given that there is no equivalent octicon and symlinked readme is rather rare, I decided to remove this and always render a book.